api/docs/http_api_index.md

```diff
@@ -1,13 +1,43 @@
 V1 API Services
 ---------------
 
+The V1 API has been removed. All previous routes can now be found under the
+[Master V3](#Master-V3-HTTP-API) API or the [CA V1](#CA-V1-HTTP-API).
```

Comment:
Hello!

1. The way they currently end up on the site sucks, and I'm gonna replace it. In fact, I'll go ahead and promise it'll be different for Puppet 4 -- if I don't have time to make the automated process better, I'll cart them over by hand. 
2. Hmm, I should probably write a confluence page or something about writing markdown that works on both docs site and github. Well, here's the capsule version:

* Fenced code blocks are cool, but you need to fence them with triple-tildes instead of triple-backticks.
* It's okay to make relative links to .markdown files, the way you would to link to a file in the same github repo. On the docs site, we munge those to .html with RedirectMatch (or nginx's equiv). 
* You can usually link to a header inside a document so it'll work on both our site and Github. We both auto-convert header text to element IDs, and while we don't use the exact same algorithm, it's close enough that they're the same for most English text. To convert a header to a linkable ID:
    * If there's a run of non-alphabetic characters at the _start_ of the header text (including numbers), delete them.
    * Delete any characters that don't match `/[a-zA-Z0-9 -]/`. (Note that underscores get the axe.)
    * Replace all spaces with hyphens. 
    * Lowercase all letters. 

So those two would be `#master-v3-http-api` and `#ca-v1-http-api`. 

I'll take a peek at the content in a sec!